scripts/makeSampleMeshFilesCuboctahedral.py

```diff
@@ -184,40 +206,88 @@ def print_to_vtu(fname, nodes, faces, index_map):
     cell_types = [42 for _ in range(ncells)]
     face_connectivity, face_offsets = translate_faces(faces, index_map)
     # Write to file
-    with open(fname, 'w') as outfile:
+    with open(fname, "w") as outfile:
```

Comment:
So many lines for a repetitive pattern. We loose the overview of the function that cannot be contained in a screen anymore. 🤷